frontend/src/app/pages/login-page/login-page.component.ts

```diff
@@ -95,9 +95,9 @@ export class LoginPageComponent implements OnInit {
     }
 
     getPasswordErrorMessage() {
-        if (this.userForm.get('password').hasError('required')) {
-            return 'Field required.';
-        }
+        return this.userForm.get('password').hasError('required') ? 'Field required.' :
```

Comment:
Please add visibility modifier (public/private/protected) and return type for `getPasswordErrorMessage` method.